xdg/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/${XDG_APPLICATION_ID}.desktop     DEST
 #install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/${XDG_APPLICATION_ID}.appdata.xml DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/metainfo)
 install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/${XDG_APPLICATION_ID}.appdata.xml DESTINATION ../share/metainfo)
 
+# TODO: check for privileges
+# If 'make install' was run without superuser privileges, the user will get a "permission denied" error
+# when 'gtk-update-icon-cache' is executed
```

Comment:
well while trying to understand the ci failure I saw in the logs it is still raising the error

Reply:
Yes, because the CI does not use a user writable CMAKE_INSTALL_PREFIX, but the default one in /usr/local. This would require to specify `-DCMAKE_INSTALL_PREFIX=/some/user/writable/dir` during cmake configuration.
Another option is to use a custom icon cache in DESTDIR by using `\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/share/icons/hicolor` in the command below. But I am not sure, what the correct behavior should be here. Maybe elevating the current output to a more prominent warning without failing is enough?